### PR TITLE
Adding local flavor for Venezuela

### PR DESCRIPTION
**Pull request by jackboot7**
**Changes**
- Added states and regions datada
- Added forms helpers for select widgets.

**All Changes**
- [x] Add an entry to the docs/changelog.rst describing the change.
- [x] Add an entry for your name in the docs/authors.rst file if it's not
    already there.
- [x] Adjust your imports to a standard form by running this command:
  
    `isort --recursive --line-width 120 localflavor tests`

**New Fields Only**
- [x] Prefix the country code to all fields.
- [x] Field names should be easily understood by developers from the target
    localflavor country. This means that English translations are usually
    not the best name unless it's for something standard like postal code,
    tax / VAT ID etc.

### CODE REVIEW

File: localflavor/ve/ve_states.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.utils.translation import ugettext_lazy as _
+
+#: A list of Venezuelan states for use as `choices` in a formfield.
+#: Data based from ISO_3166-2:VE http://en.wikipedia.org/wiki/ISO_3166-2:VE
+STATE_CHOICES = (
+    ('VE-X', _('Amazonas')),
```
Comment (benkonrath) [inline localflavor/ve/ve_states.py]: VE-X -> VE-Z

File: localflavor/ve/ve_states.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    ('VE-B', _('Anzoátegui')),
+    ('VE-C', _('Apure')),
+    ('VE-D', _('Aragua')),
+    ('VE-E', _('Barinas')),
+    ('VE-F', _('Bolívar')),
+    ('VE-G', _('Carabobo')),
+    ('VE-H', _('Cojedes')),
+    ('VE-Y', _('Delta Amacuro')),
+    ('VE-Z', _('Dependencias Federales')),
```
Comment (benkonrath) [inline localflavor/ve/ve_states.py]: VE-Z -> VE-W

File: localflavor/ve/ve_states.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    ('VE-K', _('Lara')),
+    ('VE-L', _('Mérida')),
+    ('VE-M', _('Miranda')),
+    ('VE-N', _('Monagas')),
+    ('VE-O', _('Nueva Esparta')),
+    ('VE-P', _('Portuguesa')),
+    ('VE-R', _('Sucre')),
+    ('VE-S', _('Táchira')),
+    ('VE-T', _('Trujillo')),
+    ('VE-W', _('Vargas')),
```
Comment (benkonrath) [inline localflavor/ve/ve_states.py]: VE-W -> VE-X

**Comment by jackboot7** [comment]
Hi, @benkonrath  thanks for your comments. I already fixed the data errors on my local copy based on your feedback. 

What do you mean with:

> rebase with master

.

**Comment by benkonrath** [comment]
@jackboot7 Here's a guide that found that explains how to do a rebase:
https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

You'll have to resolve the conflict in the authors file. I can understand that it might a little overwhelming so I can resolve and merge this pull request if you prefer. Just let me know.

**Comment by jackboot7** [comment]
@benkonrath  Fixed the conflicts and rebased the code. Still not sure if I did the rebasing thing the right way... Anyway, if you have any comments they'll be appreciated.

**Comment by benkonrath** [comment]
Looks good. Thanks for your contribution to django-localflavor!

**Pull request merged**